src/components/templates/Product/Progress.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import { minMedia, Progress } from '../../..';
+import { ProgressProps } from '../../molecules/Progress/Progress';
+
+interface ProductTemplateProgressProps {
+  nextStepUrl: string;
+  prevStepUrl?: string;
+  progress?: Pick<ProgressProps, 'currentStep' | 'totalSteps'>;
+  children: React.ReactNode;
+}
+
+const BackWrapper = styled.div`
+  ${minMedia.desktop`
```

Comment:
I get this error `Uncaught TypeError: Cannot read property 'desktop' of undefined
    at Module../src/components/templates/Product/Progress.tsx (Progress.tsx:14)`

Reply:
I'm working on it